blobstore/common/rpc/auditlog/auditlog_test.go

```diff
@@ -151,3 +155,110 @@ func TestBodylimit(t *testing.T) {
 		require.Equal(t, limit.expected, cfg.BodyLimit)
 	}
 }
+
+type mockMetricSender struct {
+	data []byte
+}
+
+func (m *mockMetricSender) Send(raw []byte) error {
+	m.data = append(m.data, raw...)
+	return nil
+}
+
+type mockLogFile struct {
```

Comment:
Here is `noopLogCloser` now.

Reply:
If I use `noopLogCloser`, it appears that I won't be able to capture the log `err = j.logFile.Log(auditLogJSON)` in the main code. This outcome seems unexpected.

Reply:
ok，need mock a logger if you want to cover error of function `Log`.